Control/index.js

```diff
@@ -23,6 +23,7 @@ const api = require('./lib/api.js');
 
 buildPublicConfig(config);
 
+config.http.iframeCsp = (config?.grafana?.url) ? [ config.grafana.url ] : [];
```

Comment:
You can rewrite this by using the nullish coalescing:

`config.http.iframeCsp = config.grafana.url ?? [];`

 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
actually, it might not work as I need array with this element inside

Reply:
Indeed